newIDE/app/src/EventsSheet/EventsTree/Renderers/WhileEvent.js

```diff
@@ -108,6 +110,7 @@ export default class ForEachEvent extends React.Component<
             }
             onParameterClick={this.props.onParameterClick}
             disabled={this.props.disabled}
+            renderObjectThumbnail={this.props.renderObjectThumbnail}
```

Comment:
You've edited the props that are passed to InstructionsList.
This mean that you have to:
1) Edit the typing of InstructionsList to add renderObjectThumbnail (typing is probably: `string => React.Node`. If React.Node is not found by Flow, verify that react is imported as `import * as React from 'react'`).
2) Make sure that `InstructionsList` is given `renderObjectThumbnail`everywhere where it's use (you'll get Flow errors if that's not the case)